### ISSUE TITLE
donejs done-serve broken because can-reflect

### ISSUE THREAD

**Issue by frank-dspeed** [donejs/donejs#974]
__How often can you reproduce it?__ 

- [X ] Always
- [ ] Sometimes
- [ ] Rarely
- [ ] Unable
- [ ] I didn’t try

__Description:__
```
donejs develop

> prove2@0.0.0 develop /home/frank/workspace/peep/prove2
> done-serve --develop --port 8080

done-serve starting on http://localhost:8080
Received client connection
Live-reload server listening on port 8012
TypeError: name argument is required to req.get
    at IncomingMessage.header (/home/frank/workspace/peep/prove2/node_modules/express/lib/request.js:67:11)
    at Object.getValue (file:/home/frank/workspace/peep/prove2/node_modules/can-define/define-helpers/define-helpers.js:92:19)
    at IncomingMessage.<anonymous> (file:/home/frank/workspace/peep/prove2/node_modules/can-define/define-helpers/define-helpers.js:139:29)
    at eachDefinition (file:/home/frank/workspace/peep/prove2/node_modules/can-define/map/map.js:28:11)
    at DefineMap.forEach (file:/home/frank/workspace/peep/prove2/node_modules/can-define/map/map.js:268:4)
    at Object.defineHelpers.serialize (file:/home/frank/workspace/peep/prove2/node_modules/can-define/define-helpers/define-helpers.js:128:8)
    at DefineMap.get (file:/home/frank/workspace/peep/prove2/node_modules/can-define/map/map.js:178:25)
    at Object.getValue (file:/home/frank/workspace/peep/prove2/node_modules/can-reflect/reflections/get-set/get-set.js:150:20)
    at getFastPathRoot (file:/home/frank/workspace/peep/prove2/node_modules/can-view-scope/compute_data.js:36:22)
    at ScopeKeyData.<anonymous> (file:/home/frank/workspace/peep/prove2/node_modules/can-view-scope/compute_data.js:127:22)

```

__Steps to reproduce:__
donejs add app
donejs develop

__Expected results:__
donejs develop should serve the donejs application on localhost:8080

__Actual results:__
After going on http://localhost:8080 done-serve errors with error above

__Environment:__

| Software                | Version
| ------------------ | -------
| donejs -V               | 1
| node -v                  | 7.1
| npm -v                   | 4.2
| Browser                 | newst chrome firefox
| Operating system | ubuntu 16.10

**Comment by matthewp** [comment]
As far as I can tell, all of the can-reflect stuff was done as prereleases. Are you experiencing this when using the new prereleases or just with latest?

**Comment by matthewp** [comment]
Ah, it is happening with latest, never mind.

**Comment by frank-dspeed** [comment]
Closing because fixed via can-define@1.2.1
